### PR TITLE
LPS-124241 exporting the field value as it was recorded

### PR DESCRIPTION
**Pull request by linolaoj**
Related Issue: https://issues.liferay.com/browse/LPS-124241

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @linolaoj, can you please check if these failures are related to your changes? Thanks

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ protected String getDDMFormFieldValue(
 
 		Stream<DDMFormFieldValue> stream = ddmFormFieldValues.stream();
 
-		return HtmlUtil.render(
+		return HtmlUtil.extractText(
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java:157]: @linolaoj, have you tested this solution against fields with HTML content, like Paragraph and Rich Text?

These fields use the CKEditor to input rich content, like HTML tags, and maybe the `extractText` can remove valid data from the content inputted on these fields. If this is true, then we can try to remove the break line character only.

Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java:157]: Hi @natocesarrego, I've tested with HTML content on simple text fields. I'll do some testing with Paragraph and Rich Text.
Thanks!

Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java:157]: Hi @natocesarrego , on my tests I could not see any difference of behavior for Rich text fields and paragraph other than the removal of the line breaks.
Thanks!

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java:157]: Great! Thanks for your tests, @linolaoj. I'll forward your pull. 🙂

**Pull request closed without merging**